packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -256,6 +278,25 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
         assert.strictEqual(cell2.get(), testValue);
     });
 
+    it("doesn't resend successful counter op", async function() {
+        const pendingOps = await getPendingOps(provider, true, async (c, d, map) => {
+            const counter = await d.getSharedObject<SharedCounter>(counterId);
+            counter.increment(0);
```

Comment:
i'd use a nonzero value here. If there is a bug and only the first op is re-sent, this wouldn't catch it.

Reply:
increment(0) is basically a no op, so this test won't fail if it does get submitted twice